<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddRowDialog</class>
 <widget class="QDialog" name="AddRowDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>368</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(117, 157, 255);</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3" stretch="3,1">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="5" column="0">
      <widget class="QLabel" name="igredientsLabel">
       <property name="font">
        <font>
         <family>Optima</family>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Ingredients</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QLineEdit" name="ratingLine">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(156, 157, 255)</string>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="nutritionLabel">
       <property name="font">
        <font>
         <family>Optima</family>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Nutrition</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="ingredientsLine">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(156, 157, 255)</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="cookTimeLine">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(156, 157, 255)</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLineEdit" name="directionsLine">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(156, 157, 255)</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="recipeNameLabel">
       <property name="font">
        <font>
         <family>Optima</family>
         <pointsize>15</pointsize>
         <italic>false</italic>
        </font>
       </property>
       <property name="text">
        <string>Recipe name</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QLineEdit" name="nutritionLine">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(156, 157, 255);</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="ratingLabel">
       <property name="font">
        <font>
         <family>Optima</family>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Rating</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="servingsLine">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(156, 157, 255)</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="recipeNameLine">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(156, 157, 255)</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="servingsLabel">
       <property name="font">
        <font>
         <family>Optima</family>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Servings</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="totalTimeLabel">
       <property name="font">
        <font>
         <family>Optima</family>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Total time</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="totalTimeLine">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(156, 157, 255)</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="directionsLabel">
       <property name="font">
        <font>
         <family>Optima</family>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Directions</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="prepTimeLabel">
       <property name="font">
        <font>
         <family>Optima</family>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Prep time</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="prepTimeLine">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(156, 157, 255)</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="cookTimeLabel">
       <property name="font">
        <font>
         <family>Optima</family>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Cook time</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>4</number>
     </property>
     <item alignment="Qt::AlignHCenter">
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(76, 102, 255)</string>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AddRowDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AddRowDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
